src/app/static/src/app/components/Stepper.vue

```diff
@@ -102,9 +104,12 @@
                 currentProject: state => state.currentProject,
                 projectLoading: state => state.loading
             }),
+            updatingLanguage: mapStateProp<RootState, boolean>(null,
+                (state: RootState) => state.updatingLanguage
+            ),
             ...mapGettersByNames<keyof ComputedGetters>(namespace, ["ready", "complete"]),
             loading: function () {
-                return this.loadingFromFile || !this.ready;
+                return this.loadingFromFile || this.updatingLanguage || !this.ready;
```

Comment:
At first I thought that `updatingLanguage` should be folded into `ready`, but your approach is more appropriate as `ready` relates more to progress through wizard than to global state